tests/e2e/resource_files/multus-pods.yaml

```
@@ -43,7 +43,7 @@ metadata:
             }]'
 spec:
   containers:
-  - image: praqma/network-multitool
+  - image: rancher/mirrored-library-busybox:1.37.0
```

nit: use the same busybox image as we do for other multus stuff:
https://github.com/rancher/rke2/blob/d77a569afec9f66bbe8ef72817224d6e3fec5e0b/scripts/build-images#L94
```suggestion
  - image: rancher/mirrored-library-busybox:1.36.1
```